static/js/clip.js

```diff
@@ -6,12 +6,16 @@
 (function(document, Clipboard) {
 
     var $codes = document.querySelectorAll('pre');
+    const codelangtext = document.querySelectorAll('.language-text');
 
     function addCopy(element) {
         var copy = document.createElement("button");
         copy.className = "copy";
         copy.innerText = "Copy";
         element.append(copy);
+        if (codelangtext = true) {
```

Comment:
Imperative languages are a bit confusing in this regard but a single `=` doesn't denote equity as it does in mathematics but instead _assigns_ a value to a variable. In this case this statement assigns the value `true` to the `codelangtext` variable. Essentially this means when evaluated it is equivalent to:

```ecmascript
if (true) {
  button.style.visibility="hidden";
}
```